src/csharp/Microsoft.Spark/Constants.cs

```diff
@@ -7,5 +7,6 @@ namespace Microsoft.Spark
     internal class Constants
     {
         internal const string RunningREPLEnvVar = "DOTNET_SPARK_RUNNING_REPL";
+        public const string AvroTestContainerName = "Avro Tests";
```

Comment:
Why is this part of the main library constants ?

Reply:
Good point, am moving Avro tests to its own solution and folder structure.